instructors/files/infrastructure/workshop_resources/collaborations.py

```diff
@@ -0,0 +1,19 @@
+collaborations = [
+    "Oak Alliance",
+    "Pine Partners",
+    "Maple Consortium",
+    "Cedar Coalition",
+    "Birch Brotherhood",
+    "Willow Network",
+    "Redwood Union",
+]
+
+studies = [
+    "Oak Alliance: Subset",
+    "Pine Partners: Subset",
+    "Maple Consortium: Subset",
+    "Cedar Coalition: Subset",
+    "Birch Brotherhood: Subset",
+    "Willow Network: Subset",
+    "Redwood Union: Subset",
+]
```

Comment:
If I understand correctly, the scripts will create the collaborations and only one study for each collaboration (where the study will have only the subset of 'operational' nodes).  I believe the exercise of Chapter #3 would be more interesting if each collaboration had two studies: one with  functional nodes, and the other including a non-functional ones. This way the exercise would be something like: check the status of the collaboration nodes, find which study would be ready to request a task on it. 
Otherwise, the exercise would be about deciding between requesting the task to the whole collaboration or to the study (which would be kind of obvious).
What do you think?

Reply:
Sure we can create a second study which is essentially the same as the complete collaboration - this is something that usually rather pointless but I see the educational value

I cannot give all the studies the same names (GGA2024, AGOT2024), but I could call them Maple_GGA2024 etc. Would that work?

Reply:
Sure, that would work too!

Reply:
Yes I think that's better!

Reply:
> Sure we can create a second study which is essentially the same as the complete collaboration - this is something that usually rather pointless but I see the educational value
> 
> I cannot give all the studies the same names (GGA2024, AGOT2024), but I could call them Maple_GGA2024 etc. Would that work?

I am wondering why study names should be unique? I think it is a bit of a weird requirement? I made an issue for this: https://github.com/vantage6/vantage6/issues/1466

Reply:
I added this.